### ISSUE TITLE
rows button stays strike-through when clicking on it

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#149]
```
What steps will reproduce the problem?
1. After EDIT -> Split Cells by separator
2. Cells are split correctly showing new records.
3. Using records and row buttons, to cycle between each, it sometimes gets 
stuck on records button, even though you have clicked on rows button.

What is the expected output? What do you see instead?

rows button should be released and show as active with records button showing 
strike-through.

```

Original issue reported on code.google.com by `thadguidry` on 7 Oct 2010 at 1:07

**Comment by GoogleCodeExporter** [comment]
```
Additional info that might be helpful.

When performing the steps, I did have a facet showing only starred=true for 1 
row.  The refine process shows (repeating cycle, when alternatively clicking 
rows/records buttons):

20:11:33.956 [                   refine] GET /command/core/get-history (8598ms)
20:11:33.968 [                   refine] POST /command/core/get-rows (12ms)
20:11:34.007 [                   refine] POST /command/core/compute-facets (39ms

```

Original comment by `thadguidry` on 7 Oct 2010 at 1:16

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Trunk (2.5)
```

Original comment by `thadguidry` on 10 Oct 2011 at 2:24
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 18 Sep 2012 at 3:04
- Added labels: **Milestone-2.5**
